### ISSUE TITLE
[BUG] Item sorting is always instant

### ISSUE THREAD

**Issue by f4ndor** [arkayenro/arkinventory#1563]
**What version number you are using?**
3.09.65

**What game client are you playing? Retail, Burning Crusade, Classic / Live, PTR, Beta**
Retail

**What language is the game client set to?**
English

**Describe the bug**
Latest version broke the Sorting "When" option. No matter what I choose from the dropdown ( Intant / on window open / manually ), items are always sorted instantly.

Went back to 3.09.64, the issue disappeared.

**To Reproduce**
Set the option to "on window open" and get some new item ( mail / loot / anything )

**Comment by arkayenro** [comment]
please try [30965 alpha 1](https://www.curseforge.com/wow/addons/ark-inventory/files/3697009)

**Comment by thehumandistraction** [comment]
> please try [30965 alpha 1](https://www.curseforge.com/wow/addons/ark-inventory/files/3697009)


This seems to fix both this issue and issue #1564, however search seems broken with the error:

`ArkInventory: ERROR> ...face\AddOns\ArkInventory\Core\ArkInventoryObject.lua:79: attempt to index local 'info' (a nil value)`

**Comment by arkayenro** [comment]
do you happen to have the entire trace?

i can add a sanity check to see if info is nil but it really shouldnt ever be nil so i'd like to find out where it originated.

i dont typically do mythics so i dont have a keystone to test the error with

**Comment by arkayenro** [comment]
please try [30965 alpha 2](https://www.curseforge.com/wow/addons/ark-inventory/files/3697332)

**Comment by thehumandistraction** [comment]
> do you happen to have the entire trace?
 
Sure. This seems to be the main error:

```
Message: Interface\AddOns\ArkInventory\Core\ArkInventory.lua:11464: ...face\AddOns\ArkInventory\Core\ArkInventoryObject.lua:79: attempt to index local 'info' (a nil value)
Time: Fri Mar 18 03:18:38 2022
Count: 5
Stack: Interface\AddOns\ArkInventory\Core\ArkInventory.lua:11464: ...face\AddOns\ArkInventory\Core\ArkInventoryObject.lua:79: attempt to index local 'info' (a nil value)
[string "=[C]"]: ?
[string "@Interface\AddOns\ArkInventory\Core\ArkInventory.lua"]:11464: in function `ThreadResume'
[string "*:OnUpdate"]:1: in function <[string "*:OnUpdate"]:1>

Locals: (*temporary) = "...face\AddOns\ArkInventory\Core\ArkInventoryObject.lua:79: attempt to index local 'info' (a nil value)"
```

I'll give the 2nd alpha a shot once it's available to download from CurseForge. 👍

**Comment by thehumandistraction** [comment]
Looks like that fixed it. 👍

**Comment by f4ndor** [comment]
confirming that 30965 alpha 2 has the issue fixed

**Comment by arkayenro** [comment]
fixed in 3.09.66
